main.js

```diff
@@ -0,0 +1,49 @@
+setTimeout(display, 50)
+
+if (window.location.hash === "" || window.location.hash === "#") {
+	window.location.replace("/SSE/home");
```

Comment:
Same here, don't add `/SSE`. in this case you can use `home` directly.

Reply:
That doesn't work. It brings me to https://accio1.github.io/home

Reply:
Without a `/` at the beginning. or try `./home`

Reply:
I added a base element and made it just "home". I'm assuming this is the last thing that needed to be changed, as my comment below said:

"If it's not ready, please suggest everything which needs to be changed all at once next time, as then I know I can merge as soon as I have implemented the suggestions."

If there are no other suggestions within two hours, I will assume that was the last one and merge this.

Thanks for all of your help!

Reply:
Wait!